### PR TITLE
JS - Fix mouse event names

### PR DESCRIPTION
**Pull request by calixteman**
- fix issue #12895

### CODE REVIEW

File: src/display/annotation_layer.js [hunk 1]
```diff
@@ -577,8 +579,7 @@ class WidgetAnnotationElement extends AnnotationElement {
     for (const [baseName, eventName] of names) {
       if (
         eventName === "Action" ||
-        (this.data.actions &&
-          this.data.actions[eventName.replace(" ", "")] !== undefined)
+        (this.data.actions && this.data.actions[eventName] !== undefined)
```
Comment (Snuffleupagus) [inline src/display/annotation_layer.js:582]: Would `this.data.actions?.[eventName]` suffice here?

Comment (Snuffleupagus) [inline src/display/annotation_layer.js:582]: Would is suffice to just check the following instead?
```suggestion
        this.data.actions?.[eventName]
```

**Review by timvandermeij — APPROVED** [review]

**Comment by timvandermeij** [comment]
Looks good; thanks!

**Pull request merged**